server/utils/jwt.js

```diff
@@ -1,10 +1,18 @@
-const { verify } = require('jsonwebtoken');
+const { verify, sign } = require('jsonwebtoken');
 
 const verifyToken = (token) => new Promise((resolve, reject) => {
   verify(token, process.env.SECRET_KEY, (err, match) => {
     if (err) return reject(err);
     return resolve(match);
   });
 });
+const signToken = (providerName, userId) => new Promise((resolve, reject) => {
+  sign({ providerName, providerID: userId, role: 'provider' }, process.env.SECRET_KEY, (err, token) => {
```

Comment:
```suggestion
  sign({ providerName, providerID: userId}, process.env.SECRET_KEY, (err, token) => {
```
No more need for the role here